### PR TITLE
WEB-28

### PR DESCRIPTION
**Pull request by vmleroy**

### CODE REVIEW

File: apps/eventos-web/app/events/create/page.tsx [hunk 1]
```diff
@@ -1,23 +1,25 @@
...
+    const { id: authId } = await auth.forServerComponent().auth().user().get();
+    const { id: owner_id } = await events.forServerComponent().users().auth_id(authId).get();
     const event = await events
-      .forServerAction()
+      .forServerComponent()
       .events()
       .create({ ...data, owner: { id: owner_id } });
-    // TODO: if error, show error message
     if (!event) {
+      // TODO: if error, show error message
```
Comment (vnszero) [inline apps/eventos-web/app/events/create/page.tsx:18]: we can use flash messages

File: bun.lockb
Comment (vnszero) [inline bun.lockb:1]: I think we should avoid commiting this kind of files.

File: packages/domain-auth/src/auth-cli.ts [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+              },
+            },
+          },
+        );
+      })(),
+    ),
+  forClientComponent: (): any =>
+    Auth(
+      (() => {
+        // console.log('client component', process.env.NEXT_PUBLIC_SUPABASE_URL, process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY);
```
Comment (vnszero) [inline packages/domain-auth/src/auth-cli.ts:55]: we can use flash messages here too.

File: packages/domain-auth/src/user/sign-up.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import type { User, SupabaseClient } from '@supabase/supabase-js';
+// import { headers } from 'next/headers';
```
Comment (vnszero) [inline packages/domain-auth/src/user/sign-up.ts:2]: should it stay here for future use?

**Review by vnszero — APPROVED** [review]

File: apps/eventos-web/app/events/create/page.tsx (hunk 1 above)
Comment (vmleroy) [reply apps/eventos-web/app/events/create/page.tsx:18]: Or a toast mesage

File: bun.lockb
Comment (vmleroy) [reply bun.lockb:1]: It is a good discusion of uploading this kind of .lock file. Some say it is good to maintain the same enviroment for all devs and others say it is not necessary. So i just update it when i add some new features, like a new package

File: packages/domain-auth/src/auth-cli.ts (hunk 2 above)
Comment (vmleroy) [reply packages/domain-auth/src/auth-cli.ts:55]: These console.logs where debugs, i have to remove them

File: packages/domain-auth/src/user/sign-up.ts (hunk 3 above)
Comment (vmleroy) [reply packages/domain-auth/src/user/sign-up.ts:2]: Nop, i forgot to remove. Gonna do it rn

**Review by vnszero — APPROVED** [review]

**Pull request merged**